src/rgw/rgw_process.cc

```diff
@@ -229,9 +231,19 @@ int process_request(RGWRados* const store,
     handler->put_op(op);
   rest->put_handler(handler);
 
+  string user_id = "nobody";
+  if (s->user) {
+    user_id = s->user->user_id.to_str();
+  }
+
   dout(1) << "====== req done req=" << hex << req << dec
 	  << " op status=" << op_ret
+	  << " op_name=" << op_name
+	  << " user_id=" << user_id
```

Comment:
I think this can still be bad, if we reach this via some of the early error gotos. I added another conditional similar to the one above.